app/src/chainlit.py

```diff
@@ -46,10 +45,11 @@ async def start() -> None:
             content=f"Unused URL query parameters: {query_values}",
         ).send()
 
+    user = cl.user_session.get("user")
     await cl.Message(
         author="backend",
         metadata={"engine": engine_id, "settings": settings},
-        content=f"{engine.name} started with settings:\n{pprint.pformat(settings, indent=3)}",
+        content=f"{engine.name} started {f'for {user}' if user else ''}",
```

Comment:
Reduce verbosity since the system prompt is long. Show the user's info if logged in (only relevant for deployed app).